scripts/clowder/quay.sh

```diff
@@ -7,16 +7,56 @@ if [[ -z $QUAY_ROOT ]]; then
   exit
 fi
 
+if [[ "$(uname)" == "Darwin" ]]; then
+  MAC_OS=true
+else
+  MAC_OS=false
+fi
+
+PODMANAGER="podman"
```

Comment:
I'm maybe for some other name, like IMG_MANAGER or IMAGE_BUILDER etc.

Reply:
done